### ISSUE TITLE
Animator files issue

### ISSUE THREAD

**Issue by catgod246** [Perfare/AssetStudio#226]
Was exporting a vrca file when I noticed that some of the animator files dont export. If you export all, it crashes the program and if you just upload one at a time you get an error msg on the ones that cause the issue. 

![image](https://user-images.githubusercontent.com/40562034/42752215-d085a8a4-88b2-11e8-9341-8516c23624d5.png)

**Comment by HeresHavi** [comment]
Yeah, Animator files still having tons of issues. It's new so yeah.

**Comment by Perfare** [comment]
Can you upload the file somewhere?

**Comment by catgod246** [comment]
https://www.dropbox.com/s/2rg4uhugdev7q3w/Avatar-hasan-Asset-bundle-5.6.3p.file_a26d8729-bd96-45b5-ba75-068a2d9ca5bb.1.vrca?dl=0

Does not matter which file I look at, the assets/_customavatar gives the same error. I have a few vrca files and on some other animator's  also give that error.

**Comment by HeresHavi** [comment]
Yeah, I have tried to extract Animator Files from multiple games many times. They always seem to give me errors :\

**Comment by Perfare** [comment]
Not encounter any problem in the file you provided.
